js/localization/messages/en.json

```diff
@@ -285,6 +285,7 @@
         "dxHtmlEditor-heading": "Heading",
         "dxHtmlEditor-normalText": "Normal text",
 
+        "dxFileManager-newFolderName": "Untitled folder",
```

Comment:
Can't we use "New folder", like in Windows?

Reply:
I have reviewed  Google Drive and Unix systems. The systems have "Untitled folder" default name for new directories. How do you think, we should use "New folder" name and follow for the Windows naming?

Reply:
We decided to go with "Untitled folder"

Reply:
We decide to use the "Untitled folder"